meta.js

```diff
@@ -14,13 +14,34 @@ module.exports = {
       'type': 'string',
       'required': false,
       'message': 'Project description',
-      'default': 'Nuxt.js project'
+      'default': 'A Fight for the Future website'
     },
-    author: {
+    domainName: {
+      'type': 'string',
+      'required': false,
+      'message': 'Domain name (without www)',
+      'default': 'example.com'
+    },
+    googleAnalyticsId: {
+      'type': 'string',
+      'required': false,
+      'message': 'Google Analytics ID',
+      'default': null
+    },
+    actionNetworkPetitonId: {
       'type': 'string',
-      'message': 'Author'
+      'required': false,
+      'message': 'Action Network Petition ID',
+      'default': null
     },
+    // Can't seem to make the author prompt *not* display...
+    author: {
+      'type': 'string',
+      'required': false,
+      'message': 'Author (This isn\'t used for anything. Just hit enter...)',
+    }
   },
-  completeMessage: '{{#inPlace}}To get started:\n\n  npm install # Or yarn\n  npm run dev{{else}}To get started:\n\n  cd {{destDirName}}\n  npm install # Or yarn\n  npm run dev{{/inPlace}}'
+  completeMessage: '{{#inPlace}}To get started:\n\n  npm install # Or yarn\n  npm run dev{{else}}To get started:\n\n  cd {{destDirName}}\n  npm install # Or yarn\n  npm run dev{{/inPlace}}',
+  skipInterpolation: "**/*.{vue,yml}"
```

Comment:
@shoshi This fixes the issue you were seeing with `{{ network }}` getting removed from the SocialButton. More info [here](https://github.com/vuejs/vue-cli/tree/master#skip-rendering).

Reply:
That's awesome! Thanks for tracking that down, good to know.